sdk/management/samples/src/main/java/com/azure/management/samples/DockerUtils.java

```diff
@@ -68,7 +69,8 @@ public DockerSSLConfig(String caPem, String keyPem, String certPem) {
                 sslConfig.keyStorePassword("docker");
                 sslConfig.trustStore(CertificateUtils.createTrustStore(caPem));
             } catch (Exception e) {
-                throw new DockerClientException(e.getMessage(), e);
+                ClientLogger logger = new ClientLogger(getClass());
```

Comment:
might not be the best place to define `logger`?

Reply:
I have thought logger should not add in class since it is just a sample. Done, change it to comment rather than logger.